### PR TITLE
Dependency status panel

### PR DESCRIPTION
**Pull request by xavi-**
Added a dependency status panel to make it easier to find the root cause.

Here's a screen shot of the new panel:
screen shot 2017-07-25 at 11 27 54 am

Here's a working example: https://xavi--grafana-service.int.clever.com/dashboard/script/scripted_services.js

Icons are based on this formula: `100 * 5xx / (5xx + 2xx)`.
```
>0.0% ✅  (good)
>0.1% 🤔  (nominal)
>1.0% ⚠️  (problematic)
>10.0% ❗️ (bad)
>50.0% 📛 (really-bad)

no traffic 🙊
no data (i.e. thrift) ❔
```

Everything in dependency panel is based on the last 30 minutes of data

Also upgraded the base grafana container.

Jira: https://clever.atlassian.net/browse/INFRA-2400

### CODE REVIEW

File: build_and_run_passthrough.go [hunk 1]
```diff
@@ -1,44 +0,0 @@
...
-		run = append(run, "-e", env)
-	}
-	run = append(run, "myimage")
-
-	log.Println("Starting docker container")
-	cmd = exec.Command("docker", run...)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-	panicIfErr(cmd.Run())
-}
```
Comment (nathanleiby) [inline build_and_run_passthrough.go]: ✂️

File: dashboards/scripted_services.js [hunk 2]
```diff
@@ -1577,3 +1603,254 @@ function albDashboardPanels(load_balancer, target_group) {
...
+  var period = 1800; // 30 mins in seconds
+  var end = Date.now() / 1000 | 0;
+  var start = end - period;
+
+  var loadbalancer = appData.lb_name.substr(65);
+  var targetgroup = appData.lb_target_group.substr(52);
+
+  $.ajax({
+    method: "POST",
+    url: "/api/datasources/proxy/8",
```
Comment (nathanleiby) [inline dashboards/scripted_services.js]: comment for what this proxy points to? (in case our datasources get changed)

**Comment by nathanleiby** [comment]
Nice work!

Will be excited to hear user feedback, as it gets used.

File: dashboards/scripted_services.js [hunk 3]
```diff
@@ -46,7 +52,9 @@ function getServiceRow(serviceName, lb_type) {
         "span": 12,
         "type": "text",
         "mode": "markdown",
-        "content": "## " + serviceName + " " + lb_type,
+        "content":
+          "## " + (isCanary ? "🇮🇨" : "") + " Build [" + service.build + "](" + repoUrl +
+          ") (count " + service.desired_count + ") " + "deployed at _" + deployDate + "_",
       }
```
Comment (nathanleiby) [inline dashboards/scripted_services.js]: I really like this.

image

Doesn't need to happen in this PR, but maybe a nice panel to add would be "last N deploys" for the app?

Comment (xavi-) [inline dashboards/scripted_services.js]: Hopefully we can get those deploy annotations back this sprint!

**Pull request merged**